### PR TITLE
Adds registration and landmark annotation endpoints

### PR DESCRIPTION
**Pull request by HarmvZ**
- Adds an endpoint for getting all registrations(LandmarkAnnotations) for a list of images
- Adds an endpoint for getting the OCT/obs registrations for an OCT/obs image
See #768

### CODE REVIEW

File: app/grandchallenge/retina_api/views.py [hunk 1]
```diff
@@ -776,3 +781,47 @@ def get_queryset(self):
...
+        user_id = self.kwargs["user_id"]
+        image_ids = self.request.query_params.get("image_ids")
+        if image_ids is None:
+            raise NotFound()
+        image_ids = image_ids.split(",")
+        user = get_object_or_404(get_user_model(), id=user_id)
+        queryset = user.landmarkannotationset_set.filter(
+            singlelandmarkannotation__image__id__in=image_ids
+        ).distinct()
+        # if len(queryset) == 0:
```
Comment (jmsmkn) [inline app/grandchallenge/retina_api/views.py]: Please remove the commented out code

**Comment by jmsmkn** [comment]
I think that you need to update the version of black that you're using. We've moved to 19.3b0 now. Also, I'd recommend installing the pre-commit hook which will do the formatting on your changesets automatically, would eliminate the `Run black` commits.

**Comment by HarmvZ** [comment]
> I think that you need to update the version of black that you're using. We've moved to 19.3b0 now. Also, I'd recommend installing the pre-commit hook which will do the formatting on your changesets automatically, would eliminate the `Run black` commits.

I updated it already, but I think because this branch was made before the update it still runs on the [old version](https://github.com/comic/grand-challenge.org/blob/13d4f28122d0314a57e97319c2586565fb9d89fd/Pipfile#L12) in the `push` build.

I installed the pre-commit hook now, that will indeed make things a lot easier!

**Comment by jmsmkn** [comment]
I see, thanks!

**Pull request merged**